pkg/controllers/podgroup_controller.go

```diff
@@ -123,8 +123,7 @@ func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			pgCopy.Status.Phase = schedv1alpha1.PodGroupScheduled
 		}
 
-		if pgCopy.Status.Succeeded+pgCopy.Status.Running >= pg.Spec.MinMember &&
-			pgCopy.Status.Phase == schedv1alpha1.PodGroupScheduled {
+		if pgCopy.Status.Succeeded+pgCopy.Status.Running >= pg.Spec.MinMember {
```

Comment:
I've changed this part.

In the previous code, we were only able to enter the `Running` statement when `PodGroup.Status.Phase` was set to `PodGroupScheduled`. However, in the code, only the `PostBind` logic changes `PodGroup.Status.Phase` to `PodGroupScheduled`. Due to the lack of `PostBind` logic in this test code, `PodGroup.Status.phase` cannot transition to `PodGroupRunning`.

In real cases, if the `PostBind` logic doesn't work properly, the `PodGroup` may not stop as expected. This could potentially lead to bugs.

@zwpaper, @PiotrProkop
I will appreciate your comments!

Reply:
sounds reasonable